app/hub/Views/HomeView/HomeView.scss

```diff
@@ -73,14 +73,25 @@
 	padding: 0;
 	background-color: #979797;
 }
-.HomeView__plus {
+.HomeView__upgradeContainer {
+	display: flex;
 	margin-top: 22px;
 	padding: 22px 12px;
 	background-color: #f0f3f4;
 	border-radius: 4px;
+
+	&.purple-border {
+		box-shadow: 0 0 8px 5px #920094;
+		border: solid 1.6px #920094;
+	}
```

Comment:
Let's use the `$dark-magenta` variable defined in `_colors.scss`. If you feel like converting the rest of this file to use the other predefined `_colors.scss` variables while you're at it, I certainly won't stop you.

Reply:
I am not sure why this stylesheet wasn't already using the variables - they should be available as they get pulled in through `settings_hub` in `hub.scss`.

Reply:
Definitely should be using color variables--updated!